4.0-Upgrade.md

```diff
@@ -25,6 +25,16 @@ gem 'redis-namespace'
   `concurrency + 2` connections in your pool or Sidekiq will exit.
   When in doubt, let Sidekiq size the connection pool for you.
 
+* There's a new testing API based off the `Sidekiq::Queues` namespace. All
+  assertions made against the Worker class still work as expected.
+```ruby
+assert_equal 0, Sidekiq::Queues["default"].size
+HardWorker.perform_async(:something)
```

Comment:
:something is not a legal argument since symbols aren't native JSON.  Make sure the sample code is valid and supported.